oe_theme.theme

```diff
@@ -1044,3 +1044,35 @@ function oe_theme_preprocess_language_selection_page_content(&$variables) {
     $language['path'] = $language['href'];
   }
 }
+
+/**
+ * Implements hook_preprocess_table().
+ */
+function oe_theme_preprocess_table(array &$variables): void {
+  $name_col = [];
+  foreach ($variables['header'] as &$cell) {
+    $cell['label'] = $cell['content'];
+
+    if ($cell['attributes']['colspan']) {
+      $cell['colspan'] = (string) $cell['attributes']['colspan'];
+    }
+
+    for ($i = 1; $i <= (array_key_exists('colspan', $cell) ? intval($cell['colspan']) : 1); $i++) {
+      $cell_label = (array) $cell['label'];
+      // The contents of a drupal cell can be a render array so we need to make
+      // sure we have an string to set as the 'data-ecl-table-header' property
+      // later.
+      $name_col[] = array_key_exists('#markup', $cell_label) ? $cell_label['#markup'] : $cell['label'];
```

Comment:
I am still not sure about this approach. We can't really be sure that the cell content is going to be a simple markup render array. For all we know, this may be anything so I think we need to find a more open solution. I added a possible approach here: https://github.com/openeuropa/oe_theme/pull/332/files in which I render the array and extract all tags to end up with the actual text. Let me know what you think